airbyte-integrations/connectors/source-hubspot/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY source_hubspot ./source_hubspot
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.23
+LABEL io.airbyte.version=0.2.0
```

Comment:
Why changing to 0.2.0 and not 0.1.24?

Reply:
@marcosmarxm thanks for the feedback. I bumped the minor version up since the two new streams add functionality and are not bugfixes.